app/javascript/src/components/Common/Sidebar/index.jsx

```diff
@@ -46,23 +48,34 @@ const Sidebar = () => {
   ];
 
   return (
-    <NeetoUISidebar
-      isCollapsed={isSidebarCollapsed}
-      navLinks={SIDENAV_LINKS}
-      appName={APP_NAME}
-      organizationInfo={{
-        name: "Wheel",
-        subdomain: "bigbinary.com",
-      }}
-      profileInfo={{
-        name: `${user.first_name} ${user.last_name}`,
-        imageUrl: user.profile_image_path,
-        email: user.email,
-        bottomLinks,
-      }}
-      onCollapse={() => setIsSidebarCollapsed(!isSidebarCollapsed)}
-      changelogProps={{ id: "neetochangelog-trigger" }}
-    />
+    <nav>
+      <NeetoUISidebar
+        isCollapsed={isSidebarCollapsed}
+        navLinks={SIDENAV_LINKS}
+        appName={APP_NAME}
+        organizationInfo={{
+          logo: <Logo />,
+        }}
+        profileInfo={{
+          name: `${user.first_name} ${user.last_name}`,
+          imageUrl: user.profile_image_path,
+          email: user.email,
+          bottomLinks,
+        }}
+        changelogProps={{ id: "neetochangelog-trigger" }}
+      />
+      <div className="collapse-btn flex cursor-pointer justify-center">
+        {isSidebarCollapsed ? (
+          <RightArrow
+            onClick={() => setIsSidebarCollapsed(!isSidebarCollapsed)}
+          />
+        ) : (
+          <LeftArrow
+            onClick={() => setIsSidebarCollapsed(!isSidebarCollapsed)}
+          />
+        )}
+      </div>
```

Comment:
We don't need to do the collapsed part. neetoUI latest version has removed this logic.